pkg/setting/setting_azure.go

```diff
@@ -63,5 +64,9 @@ func (cfg *Cfg) readAzureSettings() {
 		azureSettings.UserIdentityTokenEndpoint = tokenEndpointSettings
 	}
 
+	if plugins := util.SplitString(azureSection.Key("forward_settings_to_plugins").String()); len(plugins) > 0 {
```

Comment:
nit i dont think we need this `if` because `azureSettings.ForwardSettingsPlugins` is already an empty string so assigning it to an empty string doesn't matter